test/hummingbot/connector/exchange/coinzoom/test_coinzoom_api_order_book_data_source.py

```diff
@@ -0,0 +1,140 @@
+
+    @staticmethod
+    def _register_sent_request(requests_list, url, **kwargs):
+        requests_list.append((url, kwargs))
+
+    @aioresponses()
+    def test_get_last_traded_prices(self, mock_api):
+        url = f"{Constants.REST_URL}/{Constants.ENDPOINT['TICKER']}"
+        resp = {f"{self.base_asset}_{self.quote_asset}": {"last_price": 51234.56}}
+        mock_api.get(url, body=json.dumps(resp))
+
+        results = self.async_run_with_timeout(CoinzoomAPIOrderBookDataSource.get_last_traded_prices(
+            trading_pairs=[self.trading_pair],
+            throttler=self.throttler))
+
+        self.assertIn(self.trading_pair, results)
+        self.assertEqual(Decimal("51234.56"), results[self.trading_pair])
+
+    @aioresponses()
+    def test_fetch_trading_pairs(self, mock_api):
+        url = f"{Constants.REST_URL}/{Constants.ENDPOINT['SYMBOL']}"
+        resp = [{"symbol": f"{self.base_asset}/{self.quote_asset}"},
+                {"symbol": "BTC/USDT"}]
+        mock_api.get(url, body=json.dumps(resp))
+
+        results = self.async_run_with_timeout(CoinzoomAPIOrderBookDataSource.fetch_trading_pairs(
+            throttler=self.throttler))
+
+        self.assertIn(self.trading_pair, results)
+        self.assertIn("BTC-USDT", results)
+
+    @aioresponses()
+    def test_get_new_order_book(self, mock_api):
+        url = f"{Constants.REST_URL}/" \
+              f"{Constants.ENDPOINT['ORDER_BOOK'].format(trading_pair=self.base_asset+'_'+self.quote_asset)}"
+        resp = {"timestamp": 1234567899,
+                "bids": [],
+                "asks": []}
+        mock_api.get(url, body=json.dumps(resp))
+
+        order_book: CoinzoomOrderBook = self.async_run_with_timeout(
+            self.data_source.get_new_order_book(self.trading_pair))
+
+        self.assertEqual(1234567899, order_book.snapshot_uid)
+
+    @patch("websockets.connect", new_callable=AsyncMock)
+    def test_listen_for_trades(self, ws_connect_mock):
+        ws_connect_mock.return_value = self.mocking_assistant.create_websocket_mock()
+        received_messages = asyncio.Queue()
+
+        message = {"ts": [f"{self.base_asset}/{self.quote_asset}", 8772.05, 0.01, "2020-01-16T21:02:23Z"]}
+
+        self.listening_task = asyncio.get_event_loop().create_task(
+            self.data_source.listen_for_trades(ev_loop=asyncio.get_event_loop(), output=received_messages))
+
+        self.mocking_assistant.add_websocket_text_message(
+            websocket_mock=ws_connect_mock.return_value,
+            message=json.dumps(message))
+        trade_message = self.async_run_with_timeout(received_messages.get())
+
+        self.assertEqual(OrderBookMessageType.TRADE, trade_message.type)
+        self.assertEqual(int(dateparse("2020-01-16T21:02:23Z").timestamp() * 1e3), trade_message.timestamp)
+        self.assertEqual(trade_message.timestamp, trade_message.trade_id)
+        self.assertEqual(self.trading_pair, trade_message.trading_pair)
+
+    @patch("time.time")
```

Comment:
Shouldn't we, in this case, apply the same treatment as with the `asyncio.sleep` module; i.e. mock a module-specific function instead?